### PR TITLE
chore: use github actions instead of travis

### PR DESCRIPTION
**Pull request by maxigimenez**
# Pull Request Template

## Description

Use github actions instead of travis CI for a few reasons:
- [x] Keep all the setup on github
- [x] Travis CI is being deprecated so it will require a migration *(Please be aware travis-ci.org will be shutting down in several weeks, with all accounts migrating to travis-ci.com. Please stay tuned here for more information.)*
- [x] Travis CI is super slow
- [x] We are getting random fails on travis ci, let's hope gets better on github actions

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. 

## Checklist:

- [x] My code follows the style guidelines of this project. `npm run lint` passes with no errors.
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes. `npm run test` passes with no errors.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "test-prod": "NODE_ENV=production npm run unit-test && npm run e2e-test",
     "unit-test": "jest __tests__/.*.spec.js --silent",
     "e2e-test": "jest __e2e-tests__ --runInBand --silent",
-    "lint": "eslint --quiet -f codeframe src"
+    "lint": "eslint --quiet -f codeframe src",
+    "ci": "ls && npm run test-prod"
```
Comment (tnolet) [inline package.json:15]: I think we can drop the ls here, or is it for a purpose?

**Review by tnolet — COMMENTED** [review]
Just one comment

File: package.json (hunk 1 above)
Comment (maxigimenez) [reply package.json:15]: Good catch! Will remove it, was there for testing

**Review by tnolet — APPROVED** [review]
LGTM

**Pull request merged**